pkg/component/machinecontrollermanager/sidecar_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package machinecontrollermanager_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	. "github.com/gardener/gardener/pkg/component/machinecontrollermanager"
+	"github.com/gardener/gardener/pkg/utils"
+	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = It("Should return a default provider specific mcm container spec", func() {
+	container := GetSidecarContainer("test-namespace", "provider-test", "provider-test:latest")
+	Expect(container).To(BeEquivalentTo(
```

Comment:
Nit:

```suggestion
	Expect(GetSidecarContainer("test-namespace", "provider-test", "provider-test:latest")).To(BeEquivalentTo(
```

You could also define variables for the arguments and use them in your assertion below so that you don't duplicate them